06. contribute/01. introduction.md

```diff
@@ -87,12 +87,12 @@ Not all decisions can be made using lazy consensus. Issues such as those affecti
 
 ## Contribution Process
 
-* [Testing](testing)
-* [Features](features)
-* [Translations](translations)
-* [User Interface](user-interface)
-* [Documentation](documentation)
-* [Spreading the word](spreading-the-word)
+* [Testing](02. testing.md)
+* [Features](07. features.md)
+* [Translations](04. translations.md)
+* [User Interface](05. user_interface.md)
+* [Documentation](06. documentation.md)
+* [Spreading the word](06. spreading_the_word.md)
 
 
-*This document is based on the [Template For A Meritocratic Governance Document](http://www.oss-watch.ac.uk/resources/meritocraticGovernanceModel) of OSS Watch*
```

Comment:
Strange, nothing changed. I used the online github edit functionality (easier to preview the markdown), so could be because of that. Ignore ;)